README.md

```diff
@@ -142,6 +142,21 @@ static:
     management_network_address: 172.23.99.0/24
 ```
 
+Example with openstack provider:  
+You will need to install vagrant openstack provider. For more information click [here](https://github.com/ggiamarchi/vagrant-openstack-provider).  
+Do not forget to set openstack API credentials.
+To use openstack provider as default look [here](https://www.vagrantup.com/docs/providers/default.html).  
+Tip: You can disable hostmanager by using settings.yaml
+
+```
```

Comment:
You can use yaml after the backticks for syntax highlighting. It'd also be good to describe in which file this is supposed to end up.